src/job.rs

```diff
@@ -160,14 +160,22 @@ impl JobApi {
                             project_id,
                             job_id,
                             GetQueryResultsParameters {
-                                page_token,
+                                page_token: page_token.clone(),
                                 max_results: page_size,
                                 location:    Some(location.to_string()),
                                 ..Default::default()
                             },
                         )
                         .await?;
 
+                        // Waiting for completed the job.
+                        if !qr.job_complete.unwrap_or(false) {
+                            tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
It's probably best to reduce this a bit, as it will add a second of latency even for small to medium sized jobs. I don't know if BigQuery provides any guidance on this, but I would suggest something like 200ms.

Reply:
Thank you for review.
I think best solution is specify delay by user but it makes breaking changes.